### ISSUE TITLE
fix: Value binding not working in FluentSlider

### ISSUE THREAD

**Issue by greg-bishop** [microsoft/fluentui-blazor#2575]
# 🐛 Bug Report

In v4.9.0 through v4.9.3, the FluentSlider two-way binding is not working in my application.  The initial value is loaded into the slider, but when the slider is changed, the value is not updated.  I am using FluentUI Blazor within a Teams application.  I'm not sure if that's affecting the behavior or not, but it might explain why others may not have run into the issue yet.  Otherwise, I'm not sure what might be causing the issue in my environment.

## 💻 Repro or Code Sample

I have two sliders within my application that led me to discover the issue, but to verify the behavior, I used the first code sample from the FluentSlider page:

```
<div style="margin-bottom: 60px;">
    <FluentButton style="width: 90px; margin: 10px;" @onclick="() => orientation = (orientation == Orientation.Vertical) ? Orientation.Horizontal : Orientation.Vertical">Toggle orientation</FluentButton>
    <FluentSlider Label="Select a temperature" Orientation="@orientation" Min="0" Max="100" Step="10" @bind-Value=value>
        <FluentSliderLabel Position="0">0&#8451;</FluentSliderLabel>
        <FluentSliderLabel Position="10">10&#8451;</FluentSliderLabel>
        <FluentSliderLabel Position="90">90&#8451;</FluentSliderLabel>
        <FluentSliderLabel Position="100">100&#8451;</FluentSliderLabel>
    </FluentSlider>
</div>
<p>Selected value: @value</p>

@code {
    Orientation orientation = Orientation.Horizontal;
    int value = 70;
}
```

Unfortunately, providing a zipped repo isn't simple because running a custom Teams app locally requires connecting the app to your Teams account.  Let me know if there's any other info I can provide for repro.

## 🤔 Expected Behavior

When the slider is changed, the value variable should be updated.

## 😯 Current Behavior

When the slider is changed, the value variable is not updated.

![image](https://github.com/user-attachments/assets/19ecbf0b-8999-47bf-b38f-9d98d03a6118)

## 🔦 Context

The sliders I'm actually using in my application are used for adjusting the width and height of widgets within a dashboard (number of columns and rows in a CSS grid).  If needed, I could change them to dropdowns, but the FluentSlider works prior to v4.9.0.  The sliders are loaded within a FluentPopover in the real use case, but in my test with the sample FluentSlider code, I loaded them inline.

For now, I will likely continue to run on v4.7.2 because I ran into an unrelated issue with the popover in v4.8 (which is fixed in v4.9).  v4.7.2 has a bug in the FluentSortableList that is also fixed in one of the later versions, so there is incentive to be on v4.9, but v4.7.2 is the most usable version for the use cases I have right now.

## 🌍 Your Environment

* OS & Device: Windows on PC
* Browser: Google Chrome v128.0.6613.84
* .NET 8.0
* Microsoft.FluentUI.AspNetCore.Components v4.9.3
* Microsoft.TeamsFx v2.5.0

**Comment by vnbaaij** [comment]
While I understand and appreciate it is an issue on your side, you can see in the exact same demo on the demo site that the binding is indeed working.

If you can't supply reproduction code (which is understandable), I'm not sure how you expect us to be able to help you with this.

Are you perhaps running your code in SSR mode? Are you seeing errors in the browser's dev tools?

**Comment by greg-bishop** [comment]
Thanks for the quick response.  I understand it can be very difficult to debug without being able to reproduce.  It seemed worth bringing up in case you had any thoughts or were able to find anything, but it's probably too difficult to investigate.

In the simplified test case, I was using the Interactive Server render mode, though I also tried Interactive WebAssembly and Interactive Auto.  There were no errors when changing the slider.  I think there were a couple SignalR errors during page load, which I thought were unrelated, but perhaps not.

I was able to work around the issue by using Value and ValueChanged instead of bind-Value.  To add to the confusion though, after removing my workaround, the issue went away with bind-Value.  So I don't know what the issue was or if it will resurface, but if it does, hopefully the workaround will still work.  (Interestingly, this isn't the first time I've tried upgrading to v4.9.3 and run into this issue.  So I've been able to reproduce the issue - until now.)

Given the difficulty for you to reproduce, the (hopeful) workaround, and the fluctuating behavior on my end, I'm closing this issue.  Thanks again for your response and all your work.
